editor/components/button/style.scss

```diff
@@ -1,8 +1,14 @@
 .editor-button {
 	background: none;
 	border: none;
+	outline: none;
 
 	&:disabled {
 		opacity: 0.6;
 	}
+
+	&.is-active {
```

Comment:
I'm sorry, I'm tired and wasn't being very clear indeed.

My mind has been thinking about the hover state, which right now is blue text and a dark gray outline, which I'm not convinced about. I've been thinking whether we could do the inverted version for both hover, active and toggled. I.e. hover would _also_ have the dark gray background. This is in context of the accessibility and contrast discussions that took place a while back. But we might find that this feels too heavy handed, and that we might be able to have a lighter gray hover+active state, in which case only the toggled state would be dark gray.

Bottomline, probably good to not hover/active/toggled states block this PR from being merged, as this is something I will revisit across all buttons later on regardless.